_posts/2021-03-04-grafana.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "Grafana and Loki demo"
+date: 2021-03-04 09:30
+categories: meetings
+recording_url:
```

Comment:
I believe we add `recording_url` later, so we don't even need to have it here yet, but I am not sure. @b-hans ?

Reply:
Thanks for taking a look at this, Davin!  I made the suggested changes.